js/ui/handler/drag_rotate.js

```diff
@@ -73,9 +74,14 @@ class DragRotateHandler {
     disable() {
         if (!this.isEnabled()) return;
         this._el.removeEventListener('mousedown', this._onDown);
+        this._el.removeEventListener('touchstart', this._onTouchStart);
         this._enabled = false;
     }
 
+    _onTouchStart() {
+        this.disable();
```

Comment:
Yes, on devices that have both touch screen and mouse (e.g Windows Surface Book and almost all modern Windows 10 laptops), drag rotate should be enabled when mouse interaction starts.

This can be tested on  desktop chrome using the following steps:

1. Use toggle device toolbar in desktop Chrome and switch to "Nexus 6P". Long press.
2. In the same debugging session, tap on toggle device toolbar to switch back to regular desktop chrome. Dragpan rotate should work (at least after first mouse click instead of being permanently disabled).